sentences/da/homeassistant_HassGetState.yaml

```
@@ -1,4 +1,34 @@
 language: da
 intents:
   HassGetState:
-    data: []
+    data:
+      - sentences:
+          - hvad er [(status|tilstand) (på|for|i)] <name> [<i_på> <area>]
+        response: one
+        excludes_context:
+          domain:
+            - scene
+            - script
+
+      - sentences:
+          - er [(tilstand|tilstanden|status) (på|for|i)] <name> {on_off_states:state} [<i_på> <area>]
```

```suggestion
          - er [(tilstand|tilstanden|status) (på|for|i)] <navn> {on_off_states:state} [<i_på> <område>]
```

---

Removing `i` as well.

```suggestion
          - er [(tilstand|tilstanden|status) (på|for)] <navn> {on_off_states:state} [<i_på> <område>]
```